### ISSUE TITLE
[Examples] unsupported provider: Provider is not enabled

### ISSUE THREAD

**Issue by Deep-Codes** [supabase/supabase#3841]
# Bug report

> Not sure if this is an issue but still reporting it. 

## Describe the bug

Getting: `unsupported provider: Provider is not enabled` while trying live demo of [`nextjs-todolist`](https://github.com/supabase/supabase/tree/master/examples/nextjs-todo-list) in examples

## To Reproduce

Visit: https://github.com/supabase/supabase/tree/master/examples/nextjs-todo-list

Click on the live demo link in the readme: https://supabase-nextjs-todo-list.vercel.app/

Try Signing up with Google

```json
// 20211109154719
// https://ynpyqnhflnjenwlxzful.supabase.co/auth/v1/authorize?provider=google

{
  "code": 400,
  "msg": "Unsupported provider: Provider is not enabled"
}
```

## Expected behavior

Signing up should be succesful

## Screenshots

![supabase](https://user-images.githubusercontent.com/61158210/140906205-df531b89-2c4a-4ad3-80eb-bee58ecbc5ac.gif)

**Comment by bnjmnt4n** [comment]
I think the issue is that the providers are not enabled, as per https://supabase.io/docs/guides/auth/auth-google. I think the example could be updated to link to those docs as well, to avoid this confusion.

**Comment by kiwicopple** [comment]
you're right - we should actually remove these logins or add some notes to the Readme. I'll keep this open in case someone wants to help

**Comment by c4ndyfl1p** [comment]
sooooo, what do I do to make this work? facing the same issue

What do you mean by **"providers are not enabled in the live version of the demo"**?

**Comment by Hallidayo** [comment]
@kiwicopple - I've added a note to the README in the above PR to highlight the auth buttons.

**Comment by J0** [comment]
Thanks @Hallidayo !

Hopefully should be clearer with the update -- closing for now.

Joel
